lua/sketch/init.lua

```diff
@@ -74,7 +74,12 @@ M.setup = function(options)
 --- The `:Sketch new` command is used to create a new sketch.
 --- The `:Sketch run` command executes the code in the current buffer.
 ---@brief ]]
-  vim.cmd('command! -nargs=? Sketch lua require"sketch".runner(<q-args>)')
+  vim.api.nvim_create_user_command("Sketch",  function(args) M.runner(args.args) end, {
+    nargs = 1,
+    complete = function()
+      return {"new", "run"}
+    end
+  })
```

Comment:
if the `viml` syntax for user commands is preferred over the lua API (less documented), we can still implement this functionality using `vim.cmd` with the `viml` api too.
```suggestion
  vim.cmd[[
    function! s:sketchOptions(ArgLead, CmdLine, CursorPos)
      return ["new", "run"]
    endfunction
    command! -nargs=1 -complete=customlist,s:sketchOptions Sketch lua require"sketch".runner(<q-args>)
  ]]
```